org.scummvm.ScummVM.json

```diff
@@ -7,14 +7,16 @@
         "--device=all",
         "--share=ipc",
         "--share=network",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--socket=pulseaudio",
         "--filesystem=xdg-documents",
         "--filesystem=xdg-pictures"
     ],
     "command": "scummvm_wrapper",
-    "rename-icon": "scummvm",
+    "rename-icon": "org.scummvm.scummvm",
```

Comment:
By the way, we should probably consider renaming this Flatpak to `org.scummvm.scummvm` or changing the upstream ID back to an uppercase one.